docs/src/content/docs/getting-started/secrets-and-variables.mdx

```diff
@@ -2,29 +2,83 @@
 
 ## GitHub Secrets and Variables
 
 ### Environment Variables
+
 GitHub [Secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) and [Variables](https://docs.github.com/en/actions/learn-github-actions/variables) can be used to set environment variables. They are translated into environment variables in the Terrateam GitHub Action runtime environment. These environment variables may be referenced in your Terraform code.
 
-### Using TF_VAR Environment Variables
-*Secrets and variables that start with `TF_VAR_` are treated specially by Terrateam.*
+## Defining a GitHub Secret
+
+You can use the GitHub CLI or the web interface to define a GitHub secret.
+
+### Using GitHub CLI
+
+1. Open your terminal.
+2. Run the following command to create a secret named `AWS_ACCESS_KEY_ID`:
+
+    ```bash
+    gh secret set AWS_ACCESS_KEY_ID --body "AKIAIOSFODNN7EXAMPLE"
+    ```
+
+### Using GitHub Web Interface
+
+1. Access the repository with the Terrateam app installed.
+2. Navigate to **Settings** → **Secrets and variables** → **Actions**.
+    ![Step 2: Navigate to Secrets](../../../assets/navigate-secrets.png)
+3. Click on **New repository secret**.
+    ![Step 3: New repository secret](../../../assets/new-repo-secret.png)
+4. Specify the **Name** (`AWS_ACCESS_KEY_ID`) and a **Secret value** (`AKIAIOSFODNN7EXAMPLE`).
+5. Click **Add secret**.
+    ![Step 5: Add secret](../../../assets/add-secret.png)
+
+## Using GitHub Secrets in Terraform
+
+Secrets and variables that start with `TF_VAR_` are treated specially by Terrateam.
 
 GitHub secrets and variables are always uppercase, however by convention Terraform variables are lowercase. Terrateam automatically finds all secrets that start with `TF_VAR_` and creates a new environment variable that has the lowercase name. If the lowercase name exists, no action is taken. The uppercase environment variable is left unchanged. For example, the secret `TF_VAR_LOGIN_TOKEN` will create a new environment variable called `TF_VAR_login_token`.
 
+This will create an environment variable `TF_VAR_database_password` which Terraform can automatically map to a variable named `database_password` in your Terraform configuration.
+
+```hcl
+variable "database_password" {
+  description = "Password for database connection"
+  type        = string
+  sensitive   = true
+}
+
+resource "aws_db_instance" "example" {
+  # Other configuration...
+  password = var.database_password
+}
+```
 ### Security
-Your secrets and variables never touch our servers outside of Terraform plan files.
+
+GitHub secrets are stored encrypted at rest using a [libsodium sealed box](https://libsodium.gitbook.io/doc/public-key_cryptography/sealed_boxes) and are only decrypted when used in GitHub Actions workflows. 
+
+When Terrateam runs your Terraform operations:
```

Comment:
Might be worth noting that any secrets are masked when being sent as comments to the PR, etc.